pkg/cmd/pr/status/status.go

```diff
@@ -264,6 +268,10 @@ func printPrs(io *iostreams.IOStreams, totalCount int, prs ...api.PullRequest) {
 				fmt.Fprint(w, cs.Green(fmt.Sprintf("✓ %s Approved", s)))
 			}
 
+			if pr.Mergeable == "CONFLICTING" {
+				fmt.Fprintf(w, " %s", cs.Red("× Merge conflicts"))
```

Comment:
Do you think we should display if there are no merge conflicts? I feel like all the other status indicators displayed have a success status.

Reply:
I think having explicit colors and text for each of the possible states, `mergeable`, `conflicting` and `unknown` is an improvement 👍